packages/common/src/store/pages/audio-rewards/slice.ts

```diff
@@ -106,12 +106,14 @@ const slice = createSlice({
 
       if (!userChallenge) return
 
+      // Update disbursed specifiers
```

Comment:
I don't see why used to only do this for aggregate challenges

Reply:
weird